code/modules/projectiles/guns/energy/special.dm

```diff
@@ -23,7 +23,10 @@
 	force = 5
 	slot_flags = SLOT_BELT|SLOT_HOLSTER
 	charge_cost = 480
-	projectile_type = /obj/item/projectile/ion
+	projectile_type = /obj/item/projectile/ion/pistol // vorestation edit, still packs a punch but no AoE
+
+/obj/item/weapon/gun/energy/ionrifle/weak // Vorestation addition

```

Comment:
Would also be good to move the edits to a _vr file.
Could also overwrite the ion rifle weak above by doing
```c
/obj/item/weapon/gun/energy/ionrifle/weak
	projectile_type = /obj/item/projectile/ion/pistol
```

and also overwrite the ion pistol by doing

```c
 /obj/item/weapon/gun/energy/ionrifle/pistol
	projectile_type = /obj/item/projectile/ion/pistol
```
instead of modifying this file.

Reply:
Will do that when I get home from work.